From 2656b3995169294fdadbdf212ece6db01aa52c83 Mon Sep 17 00:00:00 2001
From: Lukas Bezdicka <lbezdick@redhat.com>
Date: Wed, 17 Jun 2015 14:38:23 +0200
Subject: [PATCH] [Rabbitmq] set repos_ensure to false

We should use rabbitmq only from repos provided to us by RDO, RHOS or distro.

Change-Id: I53ea9c10800846d5d0e6632b57e969159d64ff90
---
 rabbitmq/manifests/params.pp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rabbitmq/manifests/params.pp b/rabbitmq/manifests/params.pp
index 7366d95..1350d94 100644
--- a/rabbitmq/manifests/params.pp
+++ b/rabbitmq/manifests/params.pp
@@ -68,7 +68,7 @@ class rabbitmq::params {
   $management_port            = '15672'
   $package_apt_pin            = ''
   $package_gpg_key            = 'http://www.rabbitmq.com/rabbitmq-signing-key-public.asc'
-  $repos_ensure               = true
+  $repos_ensure               = false
   $manage_repos               = undef
   $service_ensure             = 'running'
   $service_manage             = true
